mkt/reviewers/views.py

```diff
@@ -918,7 +918,7 @@ def _retrieve_translation(text, language):
 
 
 @waffle_switch('reviews-translate')
-@reviewer_required
+@permission_required([('Apps', 'ModerateReview')])
```

Comment:
`` @reviewer_required(moderator=True) `` gives access to reviewers *or* moderators.  Translation is used in the moderation queue, which is inaccessible to users with only Apps:Review permission.